htdocs/bom/class/bom.class.php

```diff
@@ -396,6 +396,16 @@ public function fetchLines()
 		$this->lines = array();
 
 		$result = $this->fetchLinesCommon();
+
+		//associate to each line childbom if exist
+		if (!empty($this->lines)) {
+			foreach ($this->lines as $line) {
+				$bom_child = new BOM($this->db);
+				$res = $bom_child->fetch($line->fk_bom_child);
```

Comment:
fetch and the init of new BOM must be done only if  $line->fk_bom_child is set

Reply:
@eldy Done